README.md

```diff
@@ -55,6 +55,8 @@ Unfortunately, at this time we are not able to support the extension with [remot
 We have enabled experimental functionality to support GitHub Enterprise Server, but this feature is an experimental beta and currently unsupported. To try this out, enable the `use-enterprise` setting to authenticate with your `GitHub Enterprise Server Authentication Provider` settings
 <img width="975" alt="Use-enterprise setting checkbox" src="https://github.com/github/vscode-github-actions/assets/34719884/be76adc6-64af-47ed-84d0-627786cc4eea">
 
+We currently do not have the capability to support OS certificates or enterprise proxies, but we have plans for it in the future and it is on our backlog!
```

Comment:
Should we also mention something about the global settings in VS code itself? Customers can set proxies both on an OS level and in the app. 🤔

Reply:
How does the new wording look? I like the idea of specifying to the VS Code proxy